conf/application.conf

```diff
@@ -48,6 +48,11 @@ play.modules.enabled += "uk.gov.hmrc.play.bootstrap.HttpClientModule"
 # Json error handler
 play.http.errorHandler = "uk.gov.hmrc.organisationsmatchingapi.handlers.CustomErrorHandler"
 
+play.filters.enabled += "play.filters.cors.CORSFilter"
```

Comment:
an alternative is to use the following in the DocumentationController:
  def specification(version: String, file: String): Action[AnyContent] = {
    val as = ActorSystem()
    implicit val materializer: Materializer = Materializer(as)
    CORSActionBuilder(config).async { implicit request =>
      assets.at(s"/public/api/conf/$version", file)(request)
    }
  }
I prefer this approach because it does not apply the CORS filter globally

Reply:
Try it out @TomRafferty, though I think the Materializer is injectable, so try not to create it yourself.